examples/check-if-testcafe-active/README.MD

```diff
@@ -0,0 +1,24 @@
+# Check Whether TestCafe Is Loaded
```

Comment:
`TestCafe is loaded` is a bad term. Better to use 'opened with TestCafe`. Here and below.

Check Whether Web Page Opened With TestCafe